uktrade/great-cms
# GRTLV-332 replace notification banner with Design System notification banner component

## What
Replace the existing govuk notification banner with the Great Design System notification banner component

To test:

- Book an event and observe the success banner. 
- Cancel an event
- Contact form submission - http://cms.trade.great:8020/contact/domestic/export-support/export-enquiry-submitted/
- EYB - Expansion guide created - http://cms.trade.great:8020/international/expand-your-business-in-the-uk/guide/
- EYB - Email address confirmed
- EYB - Signed back in - about your business

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GRTLV-332

### Security

### Merging
